README.md

```diff
@@ -17,7 +17,7 @@ the `generate.sh` script.  This will create a `compile_commands.json` file at
 your workspace root. For example,
 
 ```sh
-RELEASE_VERSION=0.1
+export RELEASE_VERSION=0.1
```

Comment:
Which shell are you using where you need to export the variable.

export is only needed if you want to make the variable available inside the child processes.